envergo/moulinette/regulations/conditionnalitepac.py

```diff
@@ -44,134 +69,247 @@ class Bcae8Form(forms.Form):
-                    else:  # amenagement_dup=non
-                        result_code = "interdit_amenagement"
-                else:  # motif=transfert_parcelles
-                    result_code = "interdit_transfert_parcelles"
-        else:
-            # SI profil=autre → non_soumis
-            result_code = "non_soumis"
+                        else:
+                            result_code = "interdit_securite"
+                    elif motif == "amenagement":
+                        if amenagement_dup == "oui":
+                            result_code = "soumis_amenagement"
+                        else:
+                            result_code = "interdit_amenagement"
+                    elif motif == "amelioration_ecologique":
+                        if meilleur_emplacement == "oui":
+                            result_code = "soumis_meilleur_emplacement"
+                        else:
+                            result_code = "interdit_amelioration_ecologique"
+                    elif motif == "embellissement":
+                        result_code = "interdit_embellissement"
+                    elif motif == "autre":
+                        if motif_pac != "aucun":
+                            result_code = "soumis_autre"
+                        else:
+                            result_code = "interdit_autre"
+                elif reimplantation == "non":
+                    if motif == "amelioration_culture":
+                        if transfert_parcelles == "oui":
+                            result_code = "interdit_transfert_parcelles"
+                        else:
+                            result_code = "interdit_amelioration_culture"
+                    elif motif == "chemin_access":
+                        if lineaire_detruit_pac <= 10:
+                            result_code = "soumis_chemin_acces"
+                        else:
+                            result_code = "interdit_chemin_acces"
+                    elif motif == "securite":
+                        if motif_pac != "aucun":
+                            result_code = "soumis_autre"
+                        else:
+                            result_code = "interdit_securite"
+                    elif motif == "amenagement":
+                        if amenagement_dup == "oui":
+                            result_code = "soumis_amenagement"
+                        else:
+                            result_code = "interdit_amenagement"
+                    elif motif == "amelioration_ecologique":
+                        # X
+                        pass
+                    elif motif == "embellissement":
+                        result_code = "interdit_embellissement"
+                    elif motif == "autre":
+                        if motif_pac != "aucun":
+                            result_code = "soumis_autre"
+                        else:
+                            result_code = "interdit_autre"
 
         return result_code
```

Comment:
https://sentry.incubateur.net/organizations/betagouv/issues/134505/?environment=staging&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=3

Il y a des "vrais" cas ou `result_code` n'est pas assigné.
En regardant le code, les candidats sont soit des chemins notés ` # X pass` soit des if/elif sans else.
Donc soit la validation n'est pas assez drastique, soit, il manque des valeurs possible dans if.